cranelift-wasm/src/code_translator.rs

```diff
@@ -1099,6 +1099,33 @@ pub fn translate_operator<FE: FuncEnvironment + ?Sized>(
             let (a, b) = state.pop2();
             state.push1(builder.ins().bxor(a, b));
         }
+        Operator::I16x8Shl | Operator::I32x4Shl | Operator::I64x2Shl => {
+            let (a, b) = state.pop2();
+            let bitcast_a = optionally_bitcast_vector(a, type_of(op), builder);
+            let bitwidth = i64::from(builder.func.dfg.value_type(a).bits());
+            // the spec expects to shift with `b mod lanewidth`; so, e.g., for 16 bit lane-width
+            // we do `b AND 15`; this means fewer instructions than `iconst + urem`
```

Comment:
Nit: The comments in this file seem to be sentences with the first letter capitalized, with punctuation. Including changes below.

Reply:
Good thing @bnjbvr didn't see this since he's been hammering me in review comments about this 😄. (Honestly, I'm so used to lower-casing comments that I'm going to have to build a linter for this specific thing).